src/objects/tests/v2/test_validation.py

```diff
@@ -19,7 +19,7 @@
 
 
 @requests_mock.Mocker()
-class ObjectTypeValidationTests(TokenAuthMixin, APITestCase):
+class ObjectTypeValidationTests(TokenAuthMixin, ClearCachesMixin, APITestCase):
```

Comment:
Could you add a test to verify that creating two objects with the same type within the timeout causes only one request to be done and when creating an object after the cache key has expired the request is done again?